presto-main/src/test/java/com/facebook/presto/operator/aggregation/TestMinMaxByAggregation.java

```diff
@@ -102,6 +102,11 @@ public void testMinNull()
                 1.0,
                 createDoublesBlock(1.0, null),
                 createDoublesBlock(1.0, 2.0));
+        assertAggregation(
```

Comment:
Maybe add a test where the min/max values are not the first ones in the sequence